packages/gallery/src/components/item/itemView.js

```diff
@@ -213,15 +213,20 @@ class ItemView extends React.Component {
       { ...this.props, clickTarget },
       e
     );
-    if (
-      this.props.type === 'video' &&
-      this.props.idx === this.props.playingVideoIdx
-    ) {
-      this.props.actions.eventsListener(
-        GALLERY_CONSTS.events.VIDEO_PAUSED,
-        { ...this.props, clickTarget },
-        e
-      );
+    if (this.props.type === 'video') {
+      if (this.props.idx === this.props.playingVideoIdx) {
+        this.props.actions.eventsListener(
+          GALLERY_CONSTS.events.VIDEO_PAUSED,
+          { ...this.props, clickTarget },
+          e
+        );
+      } else {
+        this.props.actions.eventsListener(
```

Comment:
I think the click event should be handled in the `videoScrollHelper` event handler
we should decide there what to do with the click